### ISSUE TITLE
Doesn't seem to work very well on a 2080 Super (possibly also other 20-series GPUs)

### ISSUE THREAD

**Issue by TheEnhas** [Nukem9/dlssg-to-fsr3#56]
Tested 0.41 with Cyberpunk 2077 and followed all the instructions exactly.

FPS counters do pick up the extra frames but they don't seem to actually be generated, because the game isn't visually any smoother like it should be if FG is actually working. I tested it in a few different places (notably Dogtown, which is a very performance intensive area) and it still looks like it's running at its "real" FPS.

I've seen videos with people testing it on 30-series GPUs and it seems to be smooth like it should be, so I don't know if there's something wrong on my system or if 20-series GPUs just don't work for some reason.

**Comment by lionnesquaer** [comment]
Do you enabled V-Sync in nvidia control panel?

**Comment by TheEnhas** [comment]
> Do you enabled V-Sync in nvidia control panel?

Yes.

**Comment by lionnesquaer** [comment]
In that case try enable Full Screen mode in game. And tell that happens please.

**Comment by ForthtuN** [comment]
One easy way to feel it its working is by limiting game to 30 fps, its much easier to feel it then. 30 fps vs 60 with frame gen

**Comment by TheEnhas** [comment]
> In that case try enable Full Screen mode in game. And tell what happens please.

It was already running with that.

To really test frame generation I turned on Ultra RT, which normally on my system (even with DLSS) runs at 30-40 FPS due to being CPU-limited. With frame generation on it's almost double that according to the FPS counter, but it still looks like it's running at 30-40 FPS even though visually it should be 60+. So it's definitely not working like it should.

**Comment by lionnesquaer** [comment]
I'm on rtx 3060, and i can tell that i also notice strange behaviour. It like playning in not FPS like counter shown. I think to it not work like it should. But in my case it happens only in Borderless Window mode. In full screen mode game run very smooth, but have lot ohf ghosting.

**Comment by cichy45** [comment]
Do you have VRR monitor? Without is it will give issues due to not perfect sync.

**Comment by TheEnhas** [comment]
I've been watching various videos testing FSR 3 frame generation and every one of them use a 30-series GPU, which works as intended (60+ FPS _actually looks_ like 60+ FPS). For whatever reason it doesn't work for me, so 20-series GPUs might be the problem because I've done all the steps exactly, am not running any other mods, and all that's happening is that the FPS counter is being fooled.

**Comment by TheEnhas** [comment]
Just tried it again with the same results.

My normal settings is mainly the High preset with Low crowd density + DLSS Quality, which nets me an easy 60+ FPS and the image is super smooth, except in places like Dogtown where it drops into the 40-50 range. With FG on with those settings there's no visual difference but the big CPU spikes in Dogtown and the like are still there, which FG is supposed to reduce or mitigate but it's not doing so at all.

If I use the Ultra RT preset (which on average is 30-40 FPS with DLSS Quality) with FG on I get 60+ FPS on the counter but visually the image is the same as if it's running without it, much closer to 30 FPS.

So again, for me this mod simply doesn't work and I don't know why.

**Comment by TheEnhas** [comment]
My normal settings is mainly the High preset with Low crowd density + DLSS Quality, which nets me an easy 60+ FPS and the image is super smooth, except in places like Dogtown where it drops into the 40-50 range.

If I use the Ultra RT preset (which on average is 30-40 FPS with DLSS Quality) with FG on I get 60+ FPS on the counter but visually the image is the same as if it's running without it, much closer to 30 FPS.

After even more testing I've noticed that enabling frame generation at _all_ makes the image very jerky no matter the settings, and after reading posts from others + info from AMD I think that 20-series GPUs simply don't have the power to run FG properly. I can't think of any other reason why I'm not getting the results others have been getting, other than the fact they're all using 30-series GPUs.

TLDR: Don't use this unless you have a 30-series GPU.

**Comment by TheEnhas** [comment]
After testing more I've come to the conclusion that it _does_ work, but probably not as well on 20-series GPUs.. You get a bit of a boost but it's best used without ray-tracing on.
